### DISCUSSION TITLE
Malaysia Airlines Logo

### DISCUSSION THREAD

**Discussion by vanshaj-sharma** [simple-icons/simple-icons#11385] [Help with SVGs]
Hello 👋 , I tried fixing the Malaysian Airlines logo, but I am not sure if its okay. This corresponds to the issue from #11377, where I found the svg from wikimedia but the problem is the logo has 2 colour, which after removing i get the following logo. Please guide me to understand if it is correct or if I have to make changes (explain the changes as well). 
![Malaysia_Airlines_Logo (1) (1)](https://github.com/simple-icons/simple-icons/assets/68049303/d0992118-ee52-405f-8cb2-b5751ed88003)

**Comment by anhthang** [comment]
You can simply combine the red and blue colors from the icon into a single, unified color

<img width="521" alt="image" src="https://github.com/simple-icons/simple-icons/assets/10458061/b99cf15d-58b1-48c9-94db-eeb6a77f62bc">

**Reply by vanshaj-sharma** [reply]
oh, I thought we had to keep the inside part as white 🤦  
Will fix the logo and add to a pr, thank you @anhthang !!
